subsys/bluetooth/host/att.c

```diff
@@ -295,6 +297,28 @@ struct net_buf *bt_att_create_rsp_pdu(struct bt_att_chan *chan, uint8_t op);
 
 static void bt_att_sent(struct bt_l2cap_chan *ch);
 
+static void att_disconnect(struct bt_att_chan *chan)
+{
+	char addr[BT_ADDR_LE_STR_LEN];
+	int err;
+
+	/* In rare circumstances we are "forced" to disconnect the ATT bearer and the ACL.
+	 * Examples of when this is right course of action is when there is an ATT timeout, we
+	 * receive an unexpected response from the server, or the response from the server is
+	 * invalid
+	 */
+
+	bt_addr_le_to_str(bt_conn_get_dst(chan->att->conn), addr, sizeof(addr));
+	LOG_DBG("ATT disconnecting device %s", addr);
```

Comment:
@jhedberg since there is an existing `LOG_WRN` and `LOG_ERR` in the functions calling this, I think it makes sense to keep this one as `LOG_DBG`

Reply:
Fair enough. I guess this can be moved out from Draft now?

Reply:
Yub - I thought I had clicked that but I guess I didn't :D